snowshu/core/docker.py

```diff
@@ -31,24 +31,28 @@ def _create_snowshu_volume(self, volume_name: str) -> docker.models.volumes.Volu
     def convert_container_to_replica(
             self,
             replica_name: str,
-            container: docker.models.containers.Container) -> docker.models.images.Image:
+            active_container: docker.models.containers.Container,
+            passive_container: docker.models.containers.Container) -> None:
```

Comment:
We should probably return all of the images instead of None here with the change

Reply:
I have though about that, but the return is not used anywhere.